src/components/mobile-docs-sidebar/index.tsx

```diff
@@ -0,0 +1,53 @@
+import Sidebar, { SidebarProps } from 'components/sidebar'
+import { useState } from 'react'
+
+interface MobileDocsSidebar {
+  levels: SidebarProps[]
```

Comment:
💭  We could potentially include some of the `levelButtonProps` attributes into this interface, so each level can declare how its level button can render